source/documents/delete-multiple.txt

```diff
@@ -37,6 +37,16 @@ the latest |compass-short| version, see :ref:`download-install`.
 
 .. END-COMPASS-ONLY
 
+Required Roles 
+--------------
+
+To delete multiple documents, you must have access granted through one of the 
```

Comment:
Consider "have privileges granted through one... " or similar, instead of "access".